Article/Article.js

```diff
@@ -26,4 +43,68 @@ class Article {
 
 */
 
-let articles;
\ No newline at end of file
+const articles = document.querySelectorAll(".article");
+
+const articleClassify = function(content) {
+  new Article(content);
+}
+articles.forEach(articleClassify);
+
+// STRETCH LET'S DO DIS
+
+let articleBuilder = function(data){
+  const articlesDiv = document.querySelector(".articles")
+
+  // Create Article div
+  let newArticle = document.createElement("div");
+  newArticle.classList.add("article", "inProgress");
+  articlesDiv.appendChild(newArticle);
+  let newArticleDOM = document.querySelector(".inProgress")
+
+  // Add title
+  let newArticleTitle = document.createElement("h2");
+  newArticleTitle.textContent = data.title;
+  newArticleDOM.appendChild(newArticleTitle);
+
+  // Add date
+  let newArticleDate = document.createElement("p");
+  newArticleDate.classList.add("date");
+  newArticleDate.textContent = data.date;
+  newArticleDOM.appendChild(newArticleDate);
+
+  // Add content
+  data.content.forEach (function(para) {
+    if (!para == "") {
+      newArticleDOM.appendChild(document.createElement("p")).textContent = para;
+    }
+  })
+
+  
+  newArticleDOM.classList.remove("inProgress")
+
+  // Add expand button
+  let newArticleExpandButton = document.createElement("span");
+  newArticleExpandButton.classList.add("expandButton");
+  newArticleDOM.appendChild(newArticleExpandButton);
+
+  articleClassify(newArticleDOM);
+  
+}
+
+document.querySelector(".submit-button").addEventListener("click", articleSubmit)
+
+let submission = {};
+
+function articleSubmit(event) {
+  event.preventDefault();
+  submission = {};
+  const arr = document.querySelectorAll(".input-data");
+  submission["title"] = arr[0].value;
+  submission["date"] = arr[1].value;
+  submission["content"] = arr[2].value.split('\n');
+  articleBuilder(submission);
+  submission = {};
+};
```

Comment:
This whole form submission is really intense, you solved a very difficult task. As you go through this program try and think about organizational patterns. Look for how other developers tackled the same problems, try and find examples where they did it better than you.